locales/en-US/app.ftl

```diff
@@ -43,6 +43,73 @@ AppViewRouter--error-message-from-url =
 AppViewRouter--route-not-found--home =
     .specialMessage = The URL you tried to reach was not recognized.
 
+## CallNodeContextMenu
+## This is used as a context menu for the Call Tree, Flame Graph and Stack Chart
+## panels.
+
+CallNodeContextMenu--transform-merge-function = Merge function
+    .title =
+        Merging a function removes it from the profile, and assigns its time to the
+        function that called it. This happens anywhere the function was called in
+        the tree.
+CallNodeContextMenu--transform-merge-call-node = Merge node only
+    .title =
+        Merging a node removes it from the profile, and assigns its time to the
+        function's node that called it. It only removes the function from that
+        specific part of the tree. Any other places the function was called
+        will remain in the profile.
+
+# This is used as the context menu item to apply the "Focus on function" transform.
+# Variables:
+#   $isInverted (Number) - This is actually a boolean value to determine if the
+#     call tree is inverted or not. Currently there is no wayt o pass a boolean
+#     value with Localized component, that's why we are passing this as a number
+#     instead. 0 is for false and 1 is for true.
+CallNodeContextMenu--transform-focus-function =
+    { $isInverted ->
+        [0] Focus on function
+       *[other] Focus on function (inverted)
```

Comment:
This won't work. It will be interpreted as a plural, and some locales will end up with 5 variants instead of 2.

There must be a way to pass a value that is not a number? If not, we should split this in 2 different strings.

Reply:
Ah, that's unfortunate. I really wanted to keep these both in one single key, because their title is very long. But during the Matrix discussion yesterday, Gandalf said that it's also okay to use message references for that. I split the strings now and used a message reference for that title attribute. Let me know what you think!